site/content/blog/2020-06-04-zap-2-9-0-highlights/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "ZAP 2.9 Highlights"
+description: "Highlights from ZAP release 2.9.0"
+summary: "Do you know what interesting bits were added to ZAP 2.9.0? Don't read release notes? This blog post is for you! 
+Session Management Scripts, Proxy Info Display, Proxy Port Reservation Failure Handling, Options Panel(s) Filter, Active Scan Filter, and more."
+images:
+- https://www.zaproxy.org/blog/2020-06-04-zap-2-9-0-highlights/images/zapbot.png
+type: post
+tags:
+- blog
+- youdontknowzap
+date: "2020-06-04"
+addSocialPreview: true
+authors:
+    - thorin
+---
+
+Do you know what interesting bits were added to ZAP 2.9.0?
+Don't read release notes?
```

Comment:
link to the release notes?
Just noticed that you do so later, but I dont think it hurts here either.